.github/workflows/kcp-image.yaml

```diff
@@ -28,13 +28,17 @@ jobs:
         id: vars
         run: echo "::set-output name=sha_short::$(echo ${{ github.sha }} | cut -b -7)"
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
       # Build and push a KCP image, tagged with latest and the commit SHA.
       - name: Build KCP Image
         id: build-image
         uses: redhat-actions/buildah-build@v2
         with:
           image: kcp
           tags: latest ${{ steps.vars.outputs.sha_short }} ${{ github.ref_name }}
+          archs: amd64, ppc64le
```

Comment:
can we add arm64 too while we are at it?

Reply:
@sttts done 
Can u please review it again :)